src/Files.App/Shell/ContextMenu.cs

```diff
@@ -259,7 +259,7 @@ public async Task InvokeItem(int itemID)
 						{
 							cMenu2?.HandleMenuMsg((uint)User32.WindowMessage.WM_INITMENUPOPUP, (IntPtr)mii.hSubMenu, new IntPtr(ii));
 						}
-						catch (NotImplementedException)
+						catch (Exception ex) when (ex is COMException || ex is NotImplementedException)
```

Comment:
Nit:

```suggestion
						catch (Exception ex) when (ex is COMException or NotImplementedException)
```

Reply:
I see. I rewrote all similar code in the same file.